app/javascript/modules/listingDetailsLottery/ListingDetailsLotterySearchFormWithoutBuckets.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react"
```

Comment:
This entire file is a copy of `ListingDetailsLotterySearchForm.tsx`, except the `lotteryBucketDetails` prop has been removed.